src/common/turbo.ts

```diff
@@ -205,8 +210,8 @@ export class TurboAuthenticatedClient
   /**
    * Returns the current balance of the user's wallet in 'winc'.
    */
-  getBalance(): Promise<TurboBalanceResponse> {
-    return this.paymentService.getBalance();
+  getBalance(address?: NativeAddress): Promise<TurboBalanceResponse> {
```

Comment:
nit - could default this to this.signer.getNativeAddress()

```suggestion
  getBalance(address?: NativeAddress = this.signer.getNativeAddress()): Promise<TurboBalanceResponse> {
```